lib/src/platform_client.dart

```diff
@@ -411,6 +413,44 @@ class PlatformClient {
     return User.fromJson(response);
   }
 
+  //  ___ _        _          ___      _ _                ___             _   _
+  // | _ \ |_  ___| |_ ___   / __|__ _| | |___ _ _ _  _  | __|  _ _ _  __| |_(_)___ _ _  ___
+  // |  _/ ' \/ _ \  _/ _ \ | (_ / _` | | / -_) '_| || | | _| || | ' \/ _|  _| / _ \ ' \(_-<
+  // |_| |_||_\___/\__\___/  \___\__,_|_|_\___|_|  \_, | |_| \_,_|_||_\__|\__|_\___/_||_/__/
+  // Photo Gallery Functions                       |__/
+
+  Future<GalleryPhotoAggregator> listGalleryPhotosByPage(GalleryPhotoAggregator aggregator) async {
+    Map<String, dynamic> response = {};
+    if(aggregator.hasMore) {
+      response = await _httpGet('/api/smartapp/photos/$_userId', queryParameters: {'page': aggregator.nextPage.toString()});
+      List<dynamic> jsonPhotos = response['photos'];
+      return aggregator..addPage(
+        response['response']['hasMore']!,
+        jsonPhotos.map((p) => GalleryPhoto.fromJSON(p)).toList(growable: false),
+      );
+    }
+    return aggregator;
+  }
+
+  Future<List<GalleryPhoto>> listAllGalleryPhotos() async {
+    GalleryPhotoAggregator aggregator = GalleryPhotoAggregator();
+    while(aggregator.hasMore) {
+      await listGalleryPhotosByPage(aggregator);
+    }
+    return aggregator.photos;
+  }
```

Comment:
Let's remove this and force clients to use the paginated API. Some of our Explorers have hundreds of shared photos (the max I counted was 770) and I don't want to hit our API 10's of times unnecessarily.